public/ingredientMatcher.js

```diff
@@ -1,5 +1,43 @@
 $(document).ready(function(){
-	console.log(window.location);
+	$.ajax({
+		type: 'GET',
+		url: '/recipes/search/ingredients'+window.location.search,
+		success: function(response){
+			for (var i = 0; i < response.length; i++){
```

Comment:
You should make a separate function that parses the `response` and returns `html`. It will allow you to reuse it in different places and you can separate the functionality between the API interaction *(AJAX calls)* and the view rendering *(html generation)*